tests/languages/hdl/boolean_feature.test

```diff
@@ -0,0 +1,15 @@
+true
+false
+trueVal

```

Comment:
I don't think you have to test for negatives. (In keywords and numbers as well.)